src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/generate.cmd

```diff
@@ -4,7 +4,7 @@
 ::
 
 @echo off
-set autoRestVersion=0.15.0-Nightly20160212
+set autoRestVersion=0.16.0-Nightly20160310
```

Comment:
@amarzavery 
But this is very difficult to use, when it comes to composite specs, as I have to modify multiple files for testing the code generation, and then revert them back for pull requests.

Any idea how to improve this part?